### ISSUE TITLE
A 1 element box renders poorly

### ISSUE THREAD

**Issue by jeffdeville** [CSS-Tricks/MovingBoxes#24]
I'm using moving boxes as a dynamic selector.  If I put a single item in it, the image is cut off.  You can see this at:
http://apps.facebook.com/wishgenies-qa.  You'll have to click:
"Break the Cycle"
Then choose 1 recipient, and save.
That will take you to the page w/ moving boxes.

thanks guys.  Sorry to find several issues, but I'm loving this plugin.  Very cool effect.  Just what I needed.

**Comment by Mottie** [comment]
Hi Jeffdeville!

MovingBoxes wasn't really designed to work with only one box, so I'll have to dig into the code and figure out how to get it to work. Give me some time to finish up this other project :)

**Comment by jeffdeville** [comment]
Thanks mottie. I'm on vacation til next week anyway.  Take your time.  Thanks for the reply!

**Comment by Mottie** [comment]
Hi Jeffdeville!

I just updated MovingBoxes with the fix for the single panel being cut off. I also added a way to update MovingBoxes after you add or remove a panel. Check out the change log for instructions.

I reread your original post, and umm you had several issues? I couldn't get to that facebook app page to even see the problem, but I found that the right side was cut off when MovingBoxes had larger sized panels - I originally tested it on the top demo which had really small panels :P
